### DISCUSSION TITLE
Corsair RMx series PSUs

### DISCUSSION THREAD

**Discussion by No-good-deed-goes-unpunished** [EvanMulawski/FanControl.CorsairLink#81] [Ideas]
When you get a chance would it be possible to add support for the RMx series PSU from Corsair?
CP-9020200. I have some RM750x and RM850x PSUs. Thanks

**Comment by EvanMulawski** [comment]
@No-good-deed-goes-unpunished That product series is not compatible with CorsairLink/iCUE - there is no data connection to the host (e.g. USB).
